config/config.json

```diff
@@ -220,4 +220,4 @@
         "InterNodeListenAddress": ":8075",
         "InterNodeUrls": []
     }
-}
+}
```

Comment:
perfect I was wondering why every time I ran `make test` the config file was changed and I was too lazy to check the diff hehe